src/callbacks/onInit/addRRatioFilter.js

```diff
@@ -0,0 +1,8 @@
+export default function addRRatioFilter() {
```

Comment:
Any reason to do this here instead of in default settings?

Reply:
Also feel like we should make this an option probably. Just throw in`if(settings.RRatioFilter){...}`

Reply:
So this isn't actually a filter that shows up - just a setting that is passed through?

Reply:
OK, not that I finally understand the ninja webcharts-fu going on here, I'm reminded that we *really* need to add continuous filters to webcharts